packages/reports/addon/templates/components/navi-table-select-item.hbs

```diff
@@ -1,11 +1,24 @@
 {{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{option.longName}}
-{{#if option.description}}
-  {{ember-tooltip
-    text=option.description
-    side="right"
-    popperContainer="body"
-    effect="none"
-    tooltipClass="navi-table-select-tooltip"
-  }}
-{{/if}}
\ No newline at end of file
+<div
```

Comment:
Do we need to make this a tagless component and define a div?

Reply:
This is technique going forward. Glimmer components do not have a `tagName`.

> Glimmer components shift templates to be treated as "outerHTML". There is no implicit div. All the DOM elements created by the renderer are in a template.